File: app/src/main/java/vn/com/basc/inco/model/DetailTicketComponent.java
package vn.com.basc.inco.model;

import java.util.List;

/**
 * Created by SONY on 6/13/2016.
 */
public class DetailTicketComponent extends DetailBaseComponent{
    private Detail detail;

    public Detail getDetail() {
        return detail;
    }

    public class Detail{
        String id;
        String department;
        User user;
        String status;
        String type;

        public String getId() {
            return id;
        }

        public String getDepartment() {
            return department;
        }

        public User getUser() {
            return user;
        }

        public String getStatus() {
            return status;
        }

        public String getType() {
            return type;
        }
    }

}

File: Inco/DetailTicketComponent.swift
//
//  DetailTicketComponent.swift
//  Inco
//
//  Created by admin on 7/31/16.
//  Copyright © 2016 Ban Sac. All rights reserved.
//

import Foundation

class DetailTicketComponent: DetailBaseComponent {
    override init(data:NSDictionary) {
        let extend = data.valueForKey("detail") as? NSDictionary
        self.detail = DetailTicketComponent.Detail(data: extend!)
        super.init(data: data)
    }
    var  detail:DetailTicketComponent.Detail
    class Detail{
        var id:String?
        var department:String?
        var status:String?
        var type:String?
        var user:User?
        init(data:NSDictionary){
            self.id = data.valueForKey("id") as? String
            self.department = data.valueForKey("department") as? String
            self.status = data.valueForKey("status") as? String
            self.type = data.valueForKey("type") as? String
            let user = data.valueForKey("user") as? NSDictionary
            self.user = User(data: user!)
            
        }
    }
}